test/test-applications/integrations/TestApplication.CustomSdk/Program.cs

```diff
@@ -136,9 +142,9 @@ public static async Task Main(string[] args)
 
     private static async Task PingRedis(string[] args)
     {
-        var redisPort = GetRedisPort(args);
+        var redisPort = int.Parse(GetRedisPort(args));
```

Comment:
```suggestion
        var redisPort = int.Parse(GetRedisPort(args), CultureInfo.InvariantCulture);
```

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/4107/commits/e52867251269ffa2c7048f070d8e9ae01352c2e8